frontend/src/views/Enrollment/EnrollmentForm.js

```diff
@@ -0,0 +1,310 @@
+              <FormHelperText>You would like to learn and grow with help from a mentor</FormHelperText>
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('manager')}
+                helperText={(
+                  <Fragment>
+                    Your manager&rsquo;s name
+                    (<InfoLink page={MENTORING_MANA_PAGE}>how we use this</InfoLink>)
+                  </Fragment>
+                )} />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('manager_email')}
+                helperText="Your manager's email" />
+            </Grid>
+            <Grid item xs={12}>
+              <AvailabilitySelector timeAvailability={participant.time_availability} onChange={handleTimeAvailabilityChange} />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('org')}
+                label="Organization"
+                placeholder="Firefox"
+                helperText="Your Organization at Mozilla" />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('org_level')}
+                label="Organization Level"
+                placeholder="IC3"
+                helperText={(
+                  <Fragment>
+                    Your Organization Level
+                    (<InfoLink page={CLG_PAGE}>career leveling guides</InfoLink>)
+                  </Fragment>
+                )} />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('time_at_org_level')}
+                label="Time at Level"
+                select
+                helperText="Time you have been at this organizational level" >
+                {menuItems(TIMES_AT_ORG_LEVEL)}
+              </TextField>
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('org_chart_distance')}
+                label="Organizational Distance"
+                select
+                helperText="Would you like to be paired with someone close to you on the Organization Chart?" >
+                {menuItems(ORG_CHART_DISTANCES)}
+              </TextField>
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                {...textFieldProps('track_change')}
+                label="Track Change"
+                disabled={!learner}
+                required={learner}
+                select
+                helperText="Are you considering change track (such as between IC and management)?" >
+                {menuItems(TRACK_CHANGE_INTEREST)}
+              </TextField>
+            </Grid>
+            <Grid item sm={6} />
+            <Grid item xs={12} sm={6}>
+              <InterestsControl
+                title="Learner Interests"
```

Comment:
Ah, thanks for noticing that -- I hadn't thought of them as columns!  I'll switch that around.

As for hiding the control, I had initially done so, but reversed myself for a few reasons:
* I'd like people to be able to non-destructively select and de-select "mentor" and "learner".  If controls disappear, it's not clear that the underlying data is being preserved.  Especially when de-selecting both, all other controls would disappear.
* Because of the grid layout, controls move both up-and-down and left-to-right when things appear and disappear, and this can be a bit jarring

That said, the "disabled" state of Material controls is _very_subtle, and I've been confused myself filling in the form on multiple occasions.  I suspect just making this visual distinction better would help.  I'll open an issue to figure out how to address this.

(GitHub doesn't want me to reply to that comment, for whatever bizarre reasons, so I've replied here)

Reply:
After some playing around, I think I improved the disabled state quite a bit:

![image](https://user-images.githubusercontent.com/28673/107817121-b1505380-6d43-11eb-9e4b-e4a5931458a6.png)

where before the text of the control on the right was the same textPrimary color as that on the left.  I've also changed the helper text to "(learner only)" when disabled, as a tip.  Perhaps this is good enough, but if not I'm open to suggestions :)